source/Xunit.Gherkin.Quick/GherkinScenarioDefinitionExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Linq;
+
+namespace Xunit.Gherkin.Quick
+{
+    public static class GherkinScenarioDefinitionExtensions
+    {
+        public static global::Gherkin.Ast.Scenario ApplyBackground(
+            this global::Gherkin.Ast.Scenario @this, 
+            global::Gherkin.Ast.Background background)            
+        {
+            var backgroundSteps = background != null ? background.Steps : Enumerable.Empty<global::Gherkin.Ast.Step>();
```

Comment:
Let's follow "fail fast" - i.e. check for null arguments, and if null, throw ArgumentNullException.
And the caller should check for null before calling this method.